### ISSUE TITLE
Upgrade version of Node used for experiment runner builds

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#502]
**TL;DR**: Use Node 10+ for experiment runner builds.

**Narrative**
How will this help developers do their job better?

"As a (ember-lookit-frameplayer) Developer, I would like to use Node 10+ for experiment runner builds in order to use ember-cli and ember-data 3.18.0+, which have dropped support for Node 8. This makes it easier to stay current on sub-dependencies as security advisories come out."

**Acceptance Criteria**
A precise description of how/when this issue will be considered closed, e.g.:

- [x] Node 10+ used for experiment runner builds
- [x] Can successfully build using commit SHA `b9e76ea8b653e8a431d28e7278b296b1c2f499a2` (which fails due to node 8 incompatibilities currently)
- [x] Test on current master branch of e-l-f; if the upgrade causes any problems for existing code, have provided info & steps to researchers

**Implementation Notes**
Note that Node 8 is out of LTS as of December 2019 in any event. I have no attachment to 10 in particular.
